docs/user-management.md

```diff
@@ -41,6 +41,31 @@ id query parameter.
 Route **/user/admin/delete** deletes an user account. To access this route you should specify id query parameter and do
 a POST request. Be careful, you will not be able to restore deleted account.
 
+### Impersonate User / Become another user
+
+Route **/user/admin/switch** becomes an user for the current session. You need to be an administrator to use this
+feature. Place something like this in your view file to allow to jump back when being impersonated as another person:
+
+```
+if (Yii::$app->session->has(\dektrium\user\controllers\AdminController::ORIGINAL_USER_SESSION_KEY))
+    echo Html::a(
+    '<span class="glyphicon glyphicon-user"></span> Back to original user',
+     ['//user/admin/switch'], ['class' => 'btn btn-primary', 'data-method' => 'POST']);
```

Comment:
No not needed for a route AFAIR. When it's always called from the module even `admin/switch` should be enough, but this should be tested ;)